### DISCUSSION TITLE
Removed Placeholder support. Use Twig Parameters instead.

### DISCUSSION THREAD

**Discussion by claudesimo1990** [pimcore/pimcore#15622] [General]
hello, i have a problem. until now i used the replacePlaceholders method in the placeholder class to format the page content before displaying it. but the class no longer exists in the new version of pimcore. do you have an alternative?

![image](https://github.com/pimcore/pimcore/assets/45970298/b133477a-f4e3-44fb-a802-d87edc998032)

I'd like to format this kind of content

![image](https://github.com/pimcore/pimcore/assets/45970298/e4b1f43b-d57f-4f9a-8c7d-a0d3ecd64781)

**Comment by dvesh3** [comment]
Hi @claudesimo1990 Placeholder support was removed in Pimcore 10 https://github.com/pimcore/pimcore/pull/7261. you can simply output Twig params `{{ object.someMethod }}` in the template and pass the object in the params from the controller.
